seqr/views/utils/json_to_orm_utils.py

```diff
@@ -48,12 +49,12 @@ def update_family_from_json(family, json, verbose=False):
     _update_model_from_json(family, json, FAMILY_JSON_FIELD_MAP, verbose=verbose)
 
 
-def update_individual_from_json(individual, json, verbose=False, allow_unknown_keys=False):
+def update_individual_from_json(individual, json, verbose=False, allow_unknown_keys=False, save=True):
```

Comment:
what would be a reason to set save=False? seems like it's currently always =True

Reply:
I think I was testing a different approach that I backed out of and forgot this. I'll remove it